saleor/graphql/menu/mutations.py

```diff
@@ -0,0 +1,134 @@
+import graphene
+
+from ...menu import models
+from ..core.mutations import ModelDeleteMutation, ModelMutation
+
+
+class MenuInput(graphene.InputObjectType):
+    name = graphene.String(description='Name of the menu.')
+
+
+class MenuItemInput(graphene.InputObjectType):
+    menu = graphene.ID(description='Menu to which item belongs to.')
+    name = graphene.String(description='Name of the menu item.')
+    parent = graphene.ID(description='''
+        ID of the parent menu. If empty, menu will be top level
+        menu.''')
+    url = graphene.String(description='URL of the pointed item.')
+    category = graphene.ID(description='Category to which item points.')
+    collection = graphene.ID(description='Collection to which item points.')
+    page = graphene.ID(description='Page to which item points.')
+
+
+class MenuCreate(ModelMutation):
+    class Arguments:
+        input = MenuInput(
+            required=True,
+            description='Fields required to create a menu.')
```

Comment:
This description (as well as the one in "update" mutation below) needs more explanation, that only one of the four possible items will be respected as a related one.
Maybe this input should take only `url` or `id` - if ID is provided we query against the supported types of related items in the resolver or use the given `url` as a target.

Reply:
I have changed description in `MenuItemCreate/Update` - here you only need to provide menu name.